config/solr_wrapper_test.yml

```diff
@@ -1,5 +1,7 @@
 # Place any default configuration for solr_wrapper here
 port: 8985
+# TODO: Relax this when we're Solr 7-compliant
+version: 6.6.1
```

Comment:
This got fixed in some other places.  Does this restriction still make sense for Hyku?

Reply:
I've pulled in the updated solr schema from master and removed this, should pass